src/k8s-extension/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+1.5.0
+++++++++++++++++++
+* add support for extensionsType api
+* microsoft.openservicemesh: Update OSM-Arc version check for beta and CI tags
+
```

Comment:
Since there are breaking changes in this PR, please indicate them in the changlog so that customers can immdiately identify those changes.